note addition (#7035)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/security-policy-settings/interactive-logon-require-smart-card.md

> Describes the best practices, location, values, policy management and security considerations for the **Interactive logon: Require smart card** security policy setting.

Reviewer: ```suggestion
Describes the best practices, location, values, policy management, and security considerations for the **Interactive logon: Require smart card** security policy setting.
```

> [Note]

Reviewer: ```suggestion
> [!NOTE]
```

> > You may need to download the ADMX templete for your version of Windows for this policy to be applied. 

Reviewer: ```suggestion
> You may need to download the ADMX template for your version of Windows to enable this policy to be applied. 
```
